test/scripts/e2e_subs/asset-misc.sh

```diff
@@ -16,7 +16,9 @@ ACCOUNTB=$(${gcmd} account new|awk '{ print $6 }')
 ACCOUNTC=$(${gcmd} account new|awk '{ print $6 }')
 ACCOUNTD=$(${gcmd} account new|awk '{ print $6 }')
 
-${gcmd} asset create --creator ${ACCOUNT} --name asset-misc --unitname amisc --total 1000000000000
+ASSET_NAME='Birlot : décollage'
```

Comment:
The change is good, however, I would suggest pushing it to the limits : 
use the following asset name
```
Birlot : décollage vs. ࠶🦪1
```
which is 32 bytes when encoded using utf-8